doc/publishing-api-syntactic-usage.md

```diff
@@ -108,7 +114,21 @@ links a JSON Object containing arrays of links keyed by link type eg.
  - Sends the draft content item payload to the content store with a transmission timestamp to validate message ordering
  - By default, the request will discard the draft content item with a locale of 'en'
 
-Required request parameters:
+### Required request parameters:
  - `content_id` the primary identifier for the draft content item to be discarded
+
+### Optional request params:
  - `previous_version` (optional but advised) is used to ensure the request is discarding the latest version of the draft
  - `locale` (optional) is used to discard a specific draft content item where there are multiple translations (defaults to 'en')
+
+## `GET /v2/linked/:content_id`
+
+ [Request/Response detail](https://pact-broker.dev.publishing.service.gov.uk/pacts/provider/Publishing%20API/consumer/GDS%20API%20Adapters/latest#a_request_to_return_the_items_linked_to_it_given_no_content_exists)
+
+  - Retrieves all content items with a link of the specified `link_type` to the content item with given `content_id`
```

Comment:
Just a style point, I don't think we need the markup around link_type, content_id and fields here.